app/views/admin/site_customization/pages/index.html.erb

```diff
@@ -27,6 +28,7 @@
         <td><%= I18n.l page.created_at, format: :short %></td>
         <td><%= I18n.l page.created_at, format: :short %></td>
         <td><%= t("admin.site_customization.pages.page.status_#{page.status}") %></td>
+        <td><%= name_for_locale(page.locale) rescue nil %></td>
```

Comment:
What do you think about moving that `rescue nil` on to the [method definition](https://github.com/MariaCheca/consul/blob/master/app/helpers/locales_helper.rb)?